src/Meilisearch/ObjectExtensions.cs

```diff
@@ -21,5 +22,26 @@ public static class ObjectExtensions
                 propInfo => char.ToLowerInvariant(propInfo.Name[0]) + propInfo.Name.Substring(1),
                 propInfo => propInfo.GetValue(source, null).ToString());
         }
+
+        /// <summary>
+        /// Iterates through Ienumerable and removes null keys.
+        /// </summary>
+        /// <param name="objectToTransform">Object to transform.</param>
+        /// <returns>Returns the same IEnumerable with null keys removed. </returns>
```

Comment:
Can you add the documentation for the typed parameter `T`? 🙂

Reply:
Done @curquiza

Reply:
Thanks!